Assets/Scripts/SS3D/Systems/Gamemodes/Objectives/GetItemObjective.cs

```diff
@@ -21,12 +15,12 @@ public class GetItemObjective : GamemodeObjective
         /// <summary>
         /// The item id required to complete the objective.
         /// </summary>
-        [SerializeField] private ItemIDs _targetItemId;
+        [FormerlySerializedAs("_targetItemId")] [SerializeField] private Data.Enums.ItemIds _targetItemIdId;
```

Comment:
what's the FormerlySerializedAs attribute ? What does this need it ?

Reply:
Used by Unity to not loose a prefab defined value.